content/patterns/listbox/examples/listbox-collapsible.html

```diff
@@ -132,6 +136,15 @@ <h2 id="kbd_label">Keyboard Support</h2>
             </tr>
           </thead>
           <tbody>
+            <tr data-test-id="key-tab">
+              <th><kbd>Tab</kbd></th>
+              <td>
+                <ul>
+                  <li>Moves focus into and out of the listbox.</li>
+                  <li>If the listbox is expanded, selects the focused option, collapses the listbox, and moves focus out of the listbox.</li>
```

Comment:
```suggestion
                  <li>If the listbox is expanded, it selects the focused option, collapses the listbox, and moves focus out of the listbox.</li>
```

Reply:
For brevity, we don't write complete sentences in the description column of the keyboard tables unless a supplementary note is required, which is very rare. The description is a phrase that assumes an implicit subject of the key command.